modules/machine-api-capability.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * installing/cluster-capabilities.adoc
+
+:_content-type: REFERENCE
+[id="machine-api-capability_{context}"]
+= Machine API capability
+
+[discrete]
+== Purpose
+
+The `machine-api-operator`, `cluster-autoscaler-operator`, and `cluster-control-plane-machine-set-operator` Operators provide the features for the `MachineAPI` capability. You can only disable this capability if you install a cluster with user-provisioned infrastructure.
+
+The Machine API capability is responsible for all configuration and management of machines in the cluster. If you disable the Machine API capability at installation, you need to manage all nodes and machine-related tasks manually.
```

Comment:
I dunno if we want to include "nodes" here, because at that point we're tiptoeing towards the machine-config operator and kubelet, which are available regardless of whether `MachineAPI` is disabled or not.  Can we just say "... you need to manage all machine-related tasks manually"?  Or if we want more words, can we be clearer about the boundary between the roles that the user needs to cover, e.g. by linking out to [here][1]?

[1]: https://docs.openshift.com/container-platform/4.14/machine_management/user_infra/adding-compute-user-infra-general.html

Reply:
Good suggestion, and I definitely don't want more words :)